test/test-imports/python/test.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+set -e
+scriptdir=$(cd $(dirname $0) && pwd)
+
+cd $(mktemp -d)
+mkdir test && cd test
+
+cp ${scriptdir}/cdk8s.yaml  .
+cp ${scriptdir}/mattermost_crd.yaml .
+
+cdk8s import mattermost:=mattermost_crd.yaml --language python || echo "Named import not supported yet."
```

Comment:
This is working fine locally. Will try to update to get github build to work. Probably something like `2> /dev/null` should work